src/spell-kind/spells-sight.cpp

```diff
@@ -461,22 +461,26 @@ bool probing(PlayerType *player_ptr)
         }
         msg_print(nullptr);
 
-        probed_monster_info(buf, player_ptr, m_ptr, r_ptr);
-        prt(buf, 0, 0);
+        std::string buf = probed_monster_info(player_ptr, m_ptr, r_ptr);
```

Comment:
1. The name of variable is duplicated when language is English, so please change this to 'probe\_result'.
2. The type of lvalue can be 'const auto' instead of std::string because the function of right side returns std::string always.